runtime/nls/cfre/cfrerr.nls

```diff
@@ -1590,3 +1590,79 @@ J9NLS_CFR_ERR_PREVIEW_VERSION2.explanation=Classfiles with minor version 65535 m
+J9NLS_INVALID_REFERENCETYPE_DESCRIPTOR_ON_CLASS.system_action=The JVM will throw a verification or class loading-related exception such as java.lang.ClassFormatError.
+J9NLS_INVALID_REFERENCETYPE_DESCRIPTOR_ON_CLASS.user_response=Contact the provider of the class file for a corrected version.
+# END NON-TRANSLATABLE
+
+J9NLS_INVALID_REFERENCETYPE_DESCRIPTOR_ON_SUPER_CLASS=A ReferenceType descriptor cannot be referenced by a CONSTANT_Class_info structure that represents super_class
+# START NON-TRANSLATABLE
+J9NLS_INVALID_REFERENCETYPE_DESCRIPTOR_ON_SUPER_CLASS.explanation=Please consult the Java Virtual Machine Specification for a detailed explanation.
+J9NLS_INVALID_REFERENCETYPE_DESCRIPTOR_ON_SUPER_CLASS.system_action=The JVM will throw a verification or class loading-related exception such as java.lang.ClassFormatError.
+J9NLS_INVALID_REFERENCETYPE_DESCRIPTOR_ON_SUPER_CLASS.user_response=Contact the provider of the class file for a corrected version.
+# END NON-TRANSLATABLE
+
+J9NLS_INVALID_REFERENCETYPE_DESCRIPTOR_ON_INTERFACE=A ReferenceType descriptor cannot be referenced by a CONSTANT_Class_info structure that represents the interfaces array
+# START NON-TRANSLATABLE
+J9NLS_INVALID_REFERENCETYPE_DESCRIPTOR_ON_INTERFACE.explanation=Please consult the Java Virtual Machine Specification for a detailed explanation.
+J9NLS_INVALID_REFERENCETYPE_DESCRIPTOR_ON_INTERFACE.system_action=The JVM will throw a verification or class loading-related exception such as java.lang.ClassFormatError.
+J9NLS_INVALID_REFERENCETYPE_DESCRIPTOR_ON_INTERFACE.user_response=Contact the provider of the class file for a corrected version.
+# END NON-TRANSLATABLE
+
+J9NLS_CFR_ERR_CATCH_IS_REFERENCE_TYPE_DESCRIPTOR=Exception handler type cannot be a reference type descriptor
+# START NON-TRANSLATABLE
+J9NLS_CFR_ERR_CATCH_IS_REFERENCE_TYPE_DESCRIPTOR.explanation=Please consult the Java Virtual Machine Specification for a detailed explanation.
+J9NLS_CFR_ERR_CATCH_IS_REFERENCE_TYPE_DESCRIPTOR.system_action=The JVM will throw a verification or classloading-related exception such as java.lang.ClassFormatError.
+J9NLS_CFR_ERR_CATCH_IS_REFERENCE_TYPE_DESCRIPTOR.user_response=Contact the provider of the classfile for a corrected version.
+
+J9NLS_CFR_ERR_EXCEPTION_IS_REFERENCE_TYPE_DESCRIPTOR=Exception type cannot be a reference type descriptor
+# START NON-TRANSLATABLE
+J9NLS_CFR_ERR_EXCEPTION_IS_REFERENCE_TYPE_DESCRIPTOR.explanation=Please consult the Java Virtual Machine Specification for a detailed explanation.
+J9NLS_CFR_ERR_EXCEPTION_IS_REFERENCE_TYPE_DESCRIPTOR.system_action=The JVM will throw a verification or classloading-related exception such as java.lang.ClassFormatError.
+J9NLS_CFR_ERR_EXCEPTION_IS_REFERENCE_TYPE_DESCRIPTOR.user_response=Contact the provider of the classfile for a corrected version.
+
+J9NLS_CFR_ERR_INNER_CLASS_REFERENCE_TYPE=Inner class info cannot be a referencetype descriptor
```

Comment:
Better to use "The CONSTANT_Class_info at the inner_class_info_index cannot be a reference type descriptor". And add 
`# CONSTANT_Class_info is not translatable`
` # inner_class_info_index is not translatable`
above the message.